packages/plugin-haskell/__tests__/index.js

```diff
@@ -8,8 +8,14 @@ describe('haskell', () => {
 
   it('resolves links', () => {
     assert.deepEqual(Haskell.resolve(path, [target]), [
+      '{BASE_URL}/user/repo/blob/d6/src/Foo/Bar.hs',
+      '{BASE_URL}/user/repo/blob/d6/lib/Foo/Bar.hs',
+      '{BASE_URL}/user/repo/blob/d6/app/Foo/Bar.hs',
+      '{BASE_URL}/user/repo/blob/d6/test/Foo/Bar.hs',
```

Comment:
`d6` is a branch name, was this intentionally?

Reply:
I have just replicated existing tests. Will improve.